src/components/ContentTools/Content.tsx

```diff
@@ -9,58 +8,82 @@ import { MenuContext } from './Menu'
 import { useKeydownGlobal } from '@/hooks'
 import { createPortal } from 'react-dom'
 import { GutterContext } from '../GutterProvider'
+import { useTextbitSelectionBoundsState } from '../TextbitRoot'
 
 export const Content = ({ children, className }: PropsWithChildren & {
   className?: string
 }) => {
   const [isOpen, setIsOpen] = useContext(MenuContext)
-  const { offsetY, triggerSize, box } = useContext(GutterContext)
-  const innerRef = useRef<HTMLDivElement>(null)
+  const keyTriggerRef = useKeydownGlobal<HTMLDivElement>((e) => {
+    if (isOpen && (e.key === 'Escape' || e.key === 'Tab')) {
+      e.preventDefault()
+      setIsOpen(false)
+    }
+  })
+
+  return (
+    <div ref={keyTriggerRef} style={{ height: 'full' }}>
```

Comment:
`full` is not a valid value for the `height` property

Reply:
tailwind damaged